### DISCUSSION TITLE
0.74.0 Loki receiver

### DISCUSSION THREAD

**Discussion by mybigopenbox** [open-telemetry/opentelemetry-collector-contrib#20529] [Q&A] [answered]
Hi

I am using version 0.74.0

I have added a loki receiver in my config.yaml 

receivers:
  loki:
    protocols:
      http:
      grpc:
    use_incoming_timestamp: true

and the system returns

* error decoding 'receivers': unknown type: "loki" for id: "loki" (valid values: [podman_stats postgresql statsd zookeeper filelog opencensus httpcheck jmx k8s_cluster nsxt prometheus purefb awsxray googlecloudspanner windowsperfcounters signalfx sqlquery syslog cloudfoundry influxdb memcached receiver_creator udplog otlp datadog zipkin awsecscontainermetrics dotnet_diagnostics awscontainerinsightreceiver awscloudwatch sapm mongodbatlas solace windowseventlog bigip iis kafkametrics mysql otlpjsonfile prometheus_simple journald k8sobjects jaeger rabbitmq splunk_hec tcplog cloudflare collectd pulsar redis vcenter wavefront active_directory_ds elasticsearch fluentforward nginx skywalking aerospike awsfirehose carbon docker_stats expvar hostmetrics kafka kubeletstats apache azureeventhub snmp mongodb purefa sqlserver flinkmetrics googlecloudpubsub riak couchdb k8s_events])
* 
I was under the impression that the images otel/opentelemetry-collector-contrib:latest  should have container this receiver as well as others such as promtail as they are documented as such

**Comment by atoulme** [comment]
Where is it documented? The list of components we ship in our distribution are listed here: https://github.com/open-telemetry/opentelemetry-collector-releases/blob/main/distributions/otelcol-contrib/manifest.yaml

**Comment by mar4uk** [comment, marked as answer]
loki receiver will be available in the next release 0.76.0
